gatsby/src/components/SiteDistinction/SiteDistinction.module.scss

```diff
@@ -1,18 +1,23 @@
-@use '@nice-digital/nds-core/scss/core';
-@import '../../styles/settings';
+@use '../../styles/colours';
+@use '../../styles/settings';
+@use '@nice-digital/nds-core/scss/focus';
+@use '@nice-digital/nds-core/scss/media-queries';
+@use '@nice-digital/nds-core/scss/utils';
+@use '@nice-digital/nds-core/scss/spacing';
+@use '@nice-digital/nds-core/scss/typography';
 
 .wrapper {
-  border-top: core.rem(core.$nds-spacing-x-small) solid;
+  border-top: utils.rem(spacing.$x-small) solid;
   position: sticky;
   top: -0.1px; // Hack to avoid 1px gap due to some weird Chrome sub pixel rounding
   z-index: 101; // Has to be visible over the sticky section nav (z: 100) and back to top (z:99)
 
   &.bnf {
-    border-top-color: $bnf-brand-colour;
+    border-top-color: settings.$bnf-brand-colour;
   }
 
   &.bnfc {
-    border-top-color: $bnfc-brand-colour;
+    border-top-color: settings.$bnfc-brand-colour;
```

Comment:
We might as well get rid of the settings file and move into the colour tokens file?